src/Component/GeneralEntity/GeneralEntityRoot/GeneralEntityRoot.tsx

```diff
@@ -657,13 +654,8 @@ export function GeneralEntityRoot<T extends BaseEntity>({
       setSortOrder(sorter.order);
       setSortField(sorter.field as string);
     }
-
-    if (!_isNil(pagination.current)) {
-      setPageCurrent(pagination.current);
-    }
-    if (!_isNil(pagination.pageSize)) {
-      setPageCurrent(pagination.pageSize);
-    }
+    setPageCurrent(pagination.current!);
+    setPageSize(pagination.pageSize!);
```

Comment:
This was introduced by @ahennr in https://github.com/terrestris/shogun-admin/commit/10b2c179a0881f06548b4e088afa1fee2ee59fd4, I think. Are we sure your changes are fine? can @ahennr confirm?

Reply:
This work was in cooperation with @ahennr